fms_core/resources/sample_update.py

```diff
@@ -82,6 +92,22 @@ def import_field(self, field, obj, data, is_m2m=False):
                 ))
             return
 
+        if field.attribute == "volume_history_delta":
+            # Manually process volume history and don't call superclass method
+            delta_vol = blank_str_to_none(data.get("Delta Volume (uL)"))  # "" -> None for CSVs
+            if delta_vol is not None:  # Only update volume if we got a value
+                # Note: Volume history should never be None, but this prevents
+                #       a bunch of cascading tracebacks if the synthetic "id"
+                #       column created above throws a DoesNotExist error.
+                if not obj.volume_history:
+                    obj.volume_history = []
+                vol = obj.volume + Decimal(delta_vol)
+                obj.volume_history.append(create_volume_history(
+                    VolumeHistoryUpdateType.UPDATE,
+                    str(float_to_decimal(vol))
```

Comment:
sorry for the late review - float_to_decimal isnt needed here, since vol is already a decimal

Reply:
Ok i'll clean this next code update.